clientgui/res/amdicon2.xpm

```diff
@@ -0,0 +1,308 @@
+"k.	c #9D706C",
+"l.	c #720101",
+"m.	c #6C4346",
+"n.	c #CFCFD1",
+"o.	c #D3D0D3",
+"p.	c #5F1213",
+"q.	c #8B0407",
+"r.	c #BC8682",
+"s.	c #D38A8A",
+"t.	c #D67C7C",
+"u.	c #DA8685",
+"v.	c #D78C90",
+"w.	c #C15B5C",
+"x.	c #C67F80",
+"y.	c #BC8485",
+"z.	c #B37778",
+"A.	c #A97673",
+"B.	c #AF8886",
+"C.	c #710101",
+"D.	c #6D4548",
+"E.	c #D4D4D6",
+"F.	c #C5C3C5",
+"G.	c #5E0C0D",
+"H.	c #A20D0F",
+"I.	c #BC141A",
+"J.	c #CD181E",
+"K.	c #C2151C",
+"L.	c #A50E10",
+"M.	c #98080B",
+"N.	c #7E0102",
+"O.	c #6B4244",
+"P.	c #D3D3D4",
+"Q.	c #BABABA",
+"R.	c #5D0A0C",
+"S.	c #750101",
+"T.	c #8D0406",
+"U.	c #BA151A",
+"V.	c #CF1B23",
+"W.	c #CF1F27",
+"X.	c #BD2028",
+"Y.	c #B02025",
+"Z.	c #A71A1F",
+"`.	c #A01218",
+" +	c #96090C",
+".+	c #890406",
+"++	c #710100",
+"@+	c #683E3F",
+"#+	c #D1D0D1",
+"$+	c #BBBABB",
+"%+	c #5E080B",
+"&+	c #6F0709",
+"*+	c #731D1F",
+"=+	c #6D2C2F",
+"-+	c #5E3536",
+";+	c #4A3A3C",
+">+	c #413C3E",
+",+	c #3F3D40",
+"'+	c #3E3D3F",
+")+	c #433F3F",
+"!+	c #4D3C3C",
+"~+	c #573335",
+"{+	c #5F2627",
+"]+	c #641517",
+"^+	c #610304",
+"/+	c #653A3B",
+"(+	c #CECED0",
+"_+	c #C1C1C2",
+":+	c #3C1E1F",
+"<+	c #2F2425",
+"[+	c #292628",
+"}+	c #292627",
+"|+	c #504E4F",
+"1+	c #C6C6C6",
+"2+	c #A8A7A8",
+"3+	c #BEBDBE",
+"4+	c #C9C9CA",
+"5+	c #A9A8AA",
+"6+	c #999898",
+"7+	c #AAA9AA",
+"8+	c #2A2627",
+"9+	c #312222",
+"0+	c #513D40",
+"a+	c #CBCBCD",
+"b+	c #C7C8C9",
+"c+	c #292728",
+"d+	c #231F20",
+"e+	c #878586",
+"f+	c #A5A4A5",
+"g+	c #AAA8A9",
+"h+	c #6C6B6B",
+"i+	c #B6B5B6",
+"j+	c #A3A1A2",
+"k+	c #949293",
+"l+	c #929091",
+"m+	c #221F20",
+"n+	c #3F3E40",
+"o+	c #DFDFE1",
+"p+	c #717072",
+"q+	c #444244",
+"r+	c #3B393B",
+"s+	c #4A484A",
+"t+	c #5E5C5E",
+"u+	c #727171",
+"v+	c #757475",
+"w+	c #737172",
+"x+	c #696868",
+"y+	c #5F5C5E",
+"z+	c #525153",
+"A+	c #494749",
+"B+	c #423F41",
+"C+	c #3E3C3E",
+"D+	c #3C393A",
+"E+	c #383738",
+"F+	c #5A5A5C",
+"G+	c #E3E3E4",
+"H+	c #FDFDFD",
+"I+	c #E4E4E5",
+"J+	c #CDCECF",
+"K+	c #C5C7C8",
+"L+	c #CCCDCE",
+"M+	c #DBDCDE",
+"N+	c #EEF0EF",
+"O+	c #F5F7F6",
+"P+	c #F4F6F5",
+"Q+	c #EBEDED",
+"R+	c #E2E2E4",
+"S+	c #D9D9DB",
+"T+	c #D1D1D3",
+"U+	c #C7C8CA",
+"V+	c #C6C6C8",
+"W+	c #C5C5C7",
+"X+	c #E3E3E3",
+"  . + @ # $ % & * = - ; > , ' ) ! ~ { ",
+"] ^ / ( _ : < [ } | 1 2 3 4 5 6 7 8 9 ",
+"0 a b c d e f g h i j k l m n o p q r ",
+"s t u v w x y z A i j B l m n C D E F ",
+"G H I J K L M N O P Q R S T U V W X Y ",
+"Z `  ...+.@.#.$.%.&.*.=.-.;.>.,.'.).!.",
+"~.{.].^./.(._.:.<.[.}.|.1.2.3.4.5.6.7.",
+"8.9.u 0.a.b.c.d.e.f.g.h.i.j.k.l.p m.n.",
+"o.p.u q.r.s.t.u.v.w.x.y.z.A.B.C.p D.E.",
+"F.G.u 0.H.I.f g J.K.j L.M.m N.C p O.P.",
+"Q.R.S.T.d U.V.W.X.Y.Z.`. +.+N.++p @+#+",
+"$+%+&+*+=+-+;+>+,+'+,+)+!+~+{+]+^+/+(+",
+"_+:+<+[+}+|+1+2+3+4+5+6+9 7+8+}+9+0+a+",
+"b+c+d+d+d+e+f+g+h+i+j+$+k+l+d+d+m+n+7.",
+"o+p+q+r+s+t+u+v+w+x+y+z+A+B+C+D+E+F+G+",
+"H+I+J+K+L+M+N+O+P+Q+R+S+T+a+U+V+W+X+H+"};
+// This file is part of BOINC.
+// http://boinc.berkeley.edu
```

Comment:
`https`

Reply:
Then we'll have to change the old ones I copied from too.

Reply:
I was aiming for having new files use `https` and old files could be mass changed in some other PR. I could have communicated that better...